File: Estudos.NET/STATUSWS/STATUSWS/Models/Employee.cs
using System.ComponentModel.DataAnnotations;

namespace StatusWS.Models
{
    public class Employee
    {
        public int Id { get; set; }
        [Required]
        public string Name { get; set; }
        [Required]
        public string Position { get; set; }
        public string? Photo { get; set; }
        public bool IsActive { get; set; } = true;
        public EmployeeStatus Status { get; set; }

    }
}

File: EstruturaFrontend/project-teste/src/shared/types/employee.ts
export interface StatusTypeDto {
    StatusTypeId: number;
    Description: string;
}

export interface StatusDto {
    StatusId: number;
    CustomText: string | null;
    UpdateAt: string;
    DisplayText: string | null;
    StatusType: StatusTypeDto;
}

export interface EmployeeDto {
    EmployeeId: number;
    Name: string;
    Position: string;
    Photo: string;
    CreatedAt: string;
    IsActive: boolean;
    Status: StatusDto;
}